components/ImageSlider.tsx

```
@@ -0,0 +1,76 @@
+          objectFit="contain"
+          priority={currentIndex === 0}
+        />
+      </div>
+
+      {images.length > 1 && (
+        <>
+          <button
+            className="btn btn-dark position-absolute top-50 start-0 translate-middle-y m-2"
+            onClick={goToPrevious}
+            style={{ zIndex: 1 }}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)